File: Python/1 - Iniciante/1064.py
# Funcional no beecrowd
# coding: UTF-8
# Arquivo: 1064

posi = 0
soma = 0

for i in range(6):
    n = float(input())
    if n >= 0:
        posi += 1
        soma += n

print(f'{posi} valores positivos')
print(f'{(soma / posi):.1f}')

# Fim

File: go/1064.go
// Funcional no beecrowd
package main

import (
	"fmt"
)

func main() {
	posi := 0
	soma := 0.0

	for i := 0; i < 6; i++ {
		var n float64
		fmt.Scan(&n)
		if n >= 0 {
			posi++
			soma += n
		}
	}

	fmt.Printf("%d valores positivos\n%.1f\n", posi, soma/float64(posi))
}
